test/models/game_test.rb

```diff
@@ -1,7 +1,33 @@
 require 'test_helper'
 
 class GameTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    @g = Game.create(name: "New Game", white_player_id: 1, black_player_id: 1)
```

Comment:
the white and black player should be different. Might not make a difference now, but the tests could be subtly wrong if you don't represent this